src/misc/ssEconomy.lua

```diff
@@ -218,9 +228,24 @@ function ssEconomy:calculateLoanInterestRate()
     g_currentMission.missionStats.loanAnnualInterestRate = seasonsYearInterest
 end
 
+function ssEconomy:aiLoad(savegame)
+
+	-- After loading the aiVehicle, convert the vanilla pricePerMS to Seasons values (2000 to 1650 per hour), and store in a new variable, so pricePerMS can be altered without losing the original value.
+	self.originalPricePerMS = self.pricePerMS * g_seasons.economy.aiVanillaToSeasonsMultiplier;
```

Comment:
Good, maybe go with our convention of prefixing with 'ss' to prevent conflicts.
Why not store the actual original, instead of a multiplied one? If you do that it all becomes much simpler